specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/stable/2022-02-01/VirtualNetworkGatewayConnections.json

```diff
@@ -0,0 +1,275 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-02-01",
+    "title": "NetworkAdminManagementClient",
+    "description": "The Admin Network Management Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network.Admin/adminVirtualNetworkGatewayConnections": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of all virtual network gateway connections.": {
+            "$ref": "./examples/VirtualNetworkGatewayConnections/List.json"
+          }
+        },
+        "tags": [
+          "VirtualNetworkGatewayConnections"
+        ],
+        "operationId": "VirtualNetworkGatewayConnections_List",
+        "description": "Returns a list of all Virtual Network Gateway Connections.",
+        "parameters": [
+          {
+            "$ref": "../../preview/2015-06-15/Network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of virtual network gateway connections has been returned.",
+            "schema": {
+              "$ref": "#/definitions/VirtualNetworkGatewayConnectionsList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
```

Comment:
This should be
"nextLinkName": "nextLink"

Reply:
Addressed, thanks!